### PR TITLE
[IMP][8.0] Added tests to mgmtsystem_system

### PR DESCRIPTION
**Pull request by llacroix**
Added tests to up the coverage to 100%

### CODE REVIEW

**Comment by llacroix** [comment]
Not sure what's wrong with runbot.. waiting coverage

**Comment by llacroix** [comment]
Coverage 100%
https://coveralls.io/jobs/5207715

**Comment by max3903** [comment]
The form view of systems does not have labels

**Comment by llacroix** [comment]
Fixed

**Comment by bwrsandman** [comment]
:+1:

File: mgmtsystem/mgmtsystem_system.xml [hunk 1]
```diff
@@ -34,12 +36,13 @@
...
-	      groups="group_mgmtsystem_manager" 
-	      sequence="60"
-	      action="system_action"/>
+    <menuitem
+      id="menu_mgmtsystem_configuration_system"
+      name="Systems"
+      parent="menu_mgmtsystem_configuration"
+      groups="group_mgmtsystem_manager"
+      sequence="60"
+      action="system_action"/>
```
Comment (max3903) [inline mgmtsystem/mgmtsystem_system.xml:45]: Can you move this in menus.xml ?

File: mgmtsystem/mgmtsystem.xml [hunk 2]
```diff
@@ -1,70 +0,0 @@
...
-                </form>
-            </field>
-        </record>
-
-        <record model="ir.actions.act_window" id="action_system_form">
-            <field name="name">Systems</field>
-            <field name="res_model">mgmtsystem.system</field>
-            <field name="view_type">form</field>
-            <field name="view_mode">tree,form</field>
-        </record>
```
Comment (max3903) [inline mgmtsystem/mgmtsystem.xml:60]: Where did you move the window actions ?

Comment (llacroix) [inline mgmtsystem/mgmtsystem.xml:60]: They were duplicates already present in "mgmtsystem/mgmtsystem_system.xml "

File: mgmtsystem/mgmtsystem_system.xml (hunk 1 above)
Comment (llacroix) [inline mgmtsystem/mgmtsystem_system.xml:45]: No, I can't... it depends on the system_action above. It's not just a menu, it's linked to a view.

**Comment by max3903** [comment]
:+1:

**Pull request merged**